user/templates/templates.md

```diff
@@ -155,22 +155,24 @@ and also that it is not set as the default template.
 
 The procedure for uninstalling a template depends on how it was created.
 
-If the template was originaly created by cloning another template, then you can
+If the template was originally created by cloning another template, then you can
 delete it the same way as you would any other qube. In the Qube Manager,
 right-click on the template and select **Delete qube**. (If you're not sure,
 you can safely try this method first to see if it works.)
 
-If, on the other hand, the template came pre-installed or was installed by
-installing a template package in dom0, per the instructions
-[above](#installing), then you must execute the following type of command in
-dom0 in order to uninstall it:
+If, on the other hand, the template came pre-installed or was installed through
+a template package in dom0 using the instructions [above](#installing), then you
+must execute the following commands in dom0 in order to uninstall it.
 
+List the installed templates:
 ```
-$ sudo dnf remove qubes-template-<DISTRO_NAME>-<RELEASE_NUMBER>
+$ qvm-template list --installed | awk 'NR>1 {print $1}'
```

Comment:
The awk statement doesnt work as you hope to suppress the "Installed" line.
In any case, I would prefer not to use it at all - we dont want users to blindly follow code, and the output from `qvm-template list --installed` is understandable as it is.